libcxx/include/support/atomic/atomic_msvc.h

```diff
@@ -27,7 +27,37 @@
     #error Unsupported hardware
 #endif // hardware
 
-_LIBCUDACXX_BEGIN_NAMESPACE_CUDA
+_LIBCUDACXX_INLINE_VISIBILITY inline _LIBCUDACXX_CONSTEXPR int __to_gcc_order(memory_order __order) {
```

Comment:
I understand where the name comes from, but it's a bit awkward to use "gcc" in it for MSVC. We should figure out a better name for those macros than "gcc order".

Reply:
Fixed.